operator/.golangci.yaml

```diff
@@ -40,12 +40,14 @@ linters-settings:
       - empty
       - stdlib
       - Client
+      - record.EventRecorder
 issues:
   max-issues-per-linter: 0
   max-same-issues: 0
   fix: true
 output:
   sort-results: true
 run:
+  timeout: 5m
```

Comment:
Can we make the timeout run on something lower like 2 mins?

Reply:
just make sure we are on same picture, this timeout configured as a threshold value for total duration for this lint check, it's not means if some failure happens, it wait 5m before finish. so it depends on the power of the vm allocated, and the default value is `1m`, by current code base we already reach this threshold, I think here should configure a reasonable big enough time.

Reply:
I think having over 1 minute for a static code check is already unacceptable in this case. Is this normal as a duration for the codebase? Ideally I dont want to have any job running longer than 5 mins max and a lint shouldnt take that much. If its just so bad performance-wise that we have to increase it im fine

Reply:
Yes, I got your point, we can consider increase the machine power of this lint job running, currently the duration time is around 1m32s, and you can see it in the [job history ](https://status.build.kyma-project.io/job-history/gs/kyma-prow-logs/pr-logs/directory/pre-main-kyma-operator-lint), when the code base is stable enough, I think we can find a reasonable required config for this job machine.

Reply:
good news! find `concurrency` param, configured to 8 reduced the time